subset/network/network_tests.py

```diff
@@ -0,0 +1,91 @@
+import subprocess
+import time
+import sys
+
+name_of_tests = ['network.min_send', 'network.application.min_send', 'dhcp.long', 'ntp.update']
+
+arguments = sys.argv
+
+test_request = str(arguments[1])
+
+test_id = -1
```

Comment:
Why are you converting this to an integer?  All you do then is convert it back to a name later?  This seems unnecessarily convoluted.

Reply:
integer because multiple tests can be run from same code set with different args

Reply:
That says nothing about why you're converting to an integer. Nothing about your statement can't be accomplished with a meaningful semantic tag.

Reply:
hey your right, good suggestion